### PR TITLE
Feature/biz 4370

### PR DESCRIPTION
**Pull request by tigran-martirosyan**

### CODE REVIEW

File: Aims.Sdk/Aims.Sdk.nuspec [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<package >
+  <metadata>
+    <id>Aims.Sdk</id>
+    <version>$version$</version>
+    <title>Aims SDK</title>
```
Comment (daruko) [inline Aims.Sdk/Aims.Sdk.nuspec]: Aims -> AIMS

Comment (daruko) [reply Aims.Sdk/Aims.Sdk.nuspec]: Same in authors, owners, description, copyright, tags.

File: Aims.Sdk/Aims.Sdk.nuspec [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    <authors>Aims Innovation</authors>
+    <owners>Aims Innovation</owners>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>Aims SDK nuget</description>
```
Comment (daruko) [inline Aims.Sdk/Aims.Sdk.nuspec]: nuget -> NuGet

File: Aims.Sdk/Properties/AssemblyInfo.cs [hunk 3]
```diff
@@ -32,5 +31,4 @@
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("1.0.0.0")]
-[assembly: AssemblyFileVersion("1.0.0.0")]
+//[assembly: AssemblyFileVersion("1.0.0.0")]
```
Comment (daruko) [inline Aims.Sdk/Properties/AssemblyInfo.cs]: Do we need this line? Usually we don't keep code commented out unless there's a good reason.

File: .gitignore [hunk 4]
```diff
@@ -2,4 +2,6 @@
 bin
 obj
 *.suo
-*.user
\ No newline at end of file
+*.user
+.vs/
+WiX.4.0.0.5512-pre/
```
Comment (daruko) [inline .gitignore]: What's WiX.4.0.0.5512-pre? If it's supposed to be a NuGet package, I don't see any references.

**Review by daruko — CHANGES_REQUESTED** [review]
Needs some minor stylistic changes.

File: .gitignore (hunk 4 above)
Comment (tigran-martirosyan) [reply .gitignore]: It's the nuget artifacts for wix installer nuget. I'd rather it here, as the references will be added.

**Review by daruko — APPROVED** [review]

**Pull request merged**